src/components/molecules/Alert/Alert.tsx

```diff
@@ -42,23 +47,49 @@ export const Alert: React.FC<AlertProps> = ({
 }
 
 const AlertWrapper = styled.div<AlertStyledProps>`
-  --position: 2rem;
   --width: 22rem;
-  --z-index: 90;
-  --bg-color: ${(props): string => { return props.theme.globals.colors[props.type] }};
 
   display: flex;
   flex-direction: row;
-  position: fixed;
```

Comment:
maybe add the position fixed as an extra class 🤔

Reply:
Yeah maybe we can add that as a feature request for this component. I will think about it and create a ticket for that.